ui/src/app/model/auto-gen.ts

```diff
@@ -0,0 +1,8 @@
+export default interface ICreateSequence {
+    Id?: string
```

Comment:
Because Id is generated in the [backend](https://github.com/GoogleCloudPlatform/spanner-migration-tool/blob/4af47476d2d61edd9b69f8bfc8c2314f1103468f/internal/helpers.go#L66). The same interface is used for both storing existing sequences and creating new ones. At time of creation sequence Id doesn't exist.